src/node/element/path/data.rs

```diff
@@ -327,25 +346,26 @@ mod tests {
         test!("S42,0", SmoothCubicCurve, Absolute, &[42.0, 0.0]);
         test!("s \t 42,0", SmoothCubicCurve, Relative, &[42.0, 0.0]);
 
-        test!("A2.6,0 -7", EllipticalArc, Absolute, &[2.6, 0.0, -7.0]);
-        test!("a 2.6 ,0 -7", EllipticalArc, Relative, &[2.6, 0.0, -7.0]);
-
+        test!("A1 1 2.6,0 0 0 -7", EllipticalArc, Absolute, &[1.0, 1.0, 2.6, 0.0, 0.0, 0.0, -7.0]);
+        test!("a1 1 2.6,0 0 0 -7", EllipticalArc, Relative, &[1.0, 1.0, 2.6, 0.0, 0.0, 0.0, -7.0]);
+        test!("a32 32 0 00.03-45.22", EllipticalArc, Relative, &[32.0, 32.0, 0.0, 0.0, 0.0, 0.03, -45.22]);
+        
         test!("Z", Close);
         test!("z", Close);
     }
 
     #[test]
     fn parser_read_parameters() {
         macro_rules! test(
-            ($content:expr, $parameters:expr) => ({
+            ($content:expr, $is_arc:expr, $parameters:expr) => ({
```

Comment:
As before, let us create a separate function for parsing elliptical-arc parameters.

Reply:
done